searchfingerprint

```diff
@@ -40,8 +40,16 @@ while [ "${*}" != "" ] ; do
     shift
 
     #Confirm input is a video file
-    if [[ -z $(file -Ib "${INPUT}" | grep video) ]] ; then
-        echo "Input is not a video file" && continue
+    unset VIDEOCHECK
+    if [[ "$(uname -s)" = Darwin ]] ; then
+        VIDEOCHECK='file -Ib'
+    elif [[ "$(uname -s)" = Linux ]] ; then
+        VIDEOCHECK='file -i'
+    fi
+    if ! [[ -z ${VIDEOCHECK} ]] ; then
+        if [[ -z $(${VIDEOCHECK} "${INPUT}" | grep video) ]] ; then
```

Comment:
on mac running:
`VIDEOCHECK='file -Ib' ; $(${VIDEOCHECK} /Users/daverice/Desktop/test5_prores.mov)`

I get
`-bash: video/quicktime;: No such file or directory`

Reply:
isn't that because it is using command substitution and attempting to run the output of `$(${VIDEOCHECK} /Users/daverice/Desktop/test5_prores.mov)` as a command?

In the same way that just running `$(echo "hello")` will also throw an error as it attempts to execute `hello`?

On my end, using the command `VIDEOCHECK='file -i' ; $(${VIDEOCHECK} input.mov)` will also throw an error, but within the context of the script, or even something like  `VIDEOCHECK='file -i' ; var=$(${VIDEOCHECK} input.mov) ; echo "${var}"` gives the expected output/behavior.